functions.php

```diff
@@ -0,0 +1,20 @@
+<?
+/* displays list of DB info
+ *
+ * @param $winners array
+ *
+ * returns string of all winners
+ */
+function display_winners(array $winners) :string
+{
+    $output = '';
+foreach ($winners as $winner) {
```

Comment:
What happens if for some reason there are no winners in the database? Maybe some kind of error message would be nice!